src/pages/main.test.tsx

```diff
@@ -2,9 +2,42 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
 import Main from './main';
+import itemsData from 'data/items';
+import userEvent from '@testing-library/user-event';
+import { OVERLAY_ID } from 'components/modal';
+
+let fakeFetch: jest.SpyInstance<
+  Promise<Response>,
+  [input: RequestInfo | URL, init?: RequestInit | undefined]
+>;
+
+const options = {
+  headers: { Accept: 'application/json', Authorization: 'Bearer qUlM6gAOQnY1o9NArsHP' },
+  method: 'GET',
+};
+
+beforeEach(
+  () =>
+    (fakeFetch = jest.spyOn(global, 'fetch').mockImplementation(async () => {
```

Comment:
I would recommend using some sort of libs to mock fetch implementation. It will reduce this boilerplate